baiji/util/lookup.py

```diff
@@ -13,5 +13,5 @@ def get_versioned_key_remote(bucket, key_name, version_id=None):
         if e.status == 400:
             raise InvalidVersionID("Invalid versionID %s" % version_id)
         else:
-            raise e
```

Comment:
When in an except block, to reraise the original exception just say `raise`; trying `raise e` causes the exception handler to throw an exception, which is hard to debug...